boards/arm/efm32wg_stk3800/efm32wg_stk3800.dts

```diff
@@ -20,5 +24,6 @@
 
 &uart0 {
 	current-speed = <115200>;
+	loc = <1>;
```

Comment:
Do you mean we need to have a loc (location) property for the tx-pin and rx-pin separately?

Reply:
Looking at the manual, and maybe loc makes sense.  I was tracing the code and came to soc_gpio_configure/GPIO_PinModeSet.  So I'm confused about what location means for which pins are used.

Reply:
The loc is used to instruct which pin (ball) is going to be used. As a consequence, we need to pass which port will be used and the alternate configuration (location). Take a look at the section "6.7 Alternate Functionality Overview" in this document:
https://www.silabs.com/documents/public/data-sheets/efr32fg1-datasheet.pdf

Reply:
@galak,

What do you say?

Take a look at the function `uart_gecko_init_pins` in **drivers/serial/uart_gecko.c** file to understand how the "loc" property is used. If you want we can slipt the loc for rx and tx because they can be eventually different.

Reply:
For the EFM32WG (and all the older EFM32 i think), there is only one loc per U(S)ART, see e.g. https://www.silabs.com/documents/public/reference-manuals/EFM32WG-RM.pdf chapter 17.5.22 USARTn_ROUTE - I/O Routing Register.

This should be considered (one loc per peripheral vs. one loc per pin in a peripheral), if the driver implementation is changed.